backstack/build.gradle.kts

```diff
@@ -30,24 +34,22 @@ kotlin {
         implementation(libs.compose.runtime.saveable)
       }
     }
-    maybeCreate("androidMain").apply {
+    val androidMain by getting {
       dependencies {
         implementation(libs.androidx.lifecycle.viewModel.compose)
         api(libs.androidx.lifecycle.viewModel)
         api(libs.androidx.compose.runtime)
       }
     }
-    maybeCreate("commonTest").apply { dependencies { implementation(libs.kotlin.test) } }
+    val commonTest by getting { dependencies { implementation(libs.kotlin.test) } }
     val commonJvmTest =
       maybeCreate("commonJvmTest").apply {
         dependencies {
           implementation(libs.junit)
           implementation(libs.truth)
         }
       }
-    maybeCreate("jvmTest").apply { dependsOn(commonJvmTest) }
-    val iosMain by getting
-    val iosSimulatorArm64Main by getting { dependsOn(iosMain) }
```

Comment:
Is this no longer necessary with this change? From [here](https://kotlinlang.org/docs/multiplatform-hierarchy.html#target-shortcuts-and-arm64-apple-silicon-simulators) it seems like this is still necessary to make the simulator targets depend on iOS.